cpanfile

```diff
@@ -58,19 +98,32 @@ requires 'Test::Synopsis';
 requires 'Test::Version';
 requires 'Test::Warnings';
 
+requires_by_perl 'Devel::Cover',
+	before 5.010 => do_not_install,
+	before 5.012 => '==1.42',
+	;
+
+requires_by_perl 'Test::MockModule',
+	before 5.012 => '==0.178',
+	;
+
+requires_by_perl 'Test2::Harness',
+	before 5.010 => do_not_install,
+	before 5.012 => '==1.000156',
```

Comment:
I wonder if this should not be `before 5.013` there

Reply:
Yes, should be `5.014`. There was an index issue with JUnit renderer, discussed yesterday on IRC `#toolchain` channel. Should be fixed now (at least didn't fail .

This commit was to test `5.010`.